source/downloads.html

```diff
@@ -8,21 +8,80 @@ <h1>Download Apache SDAP (incubating) releases</h1>
 <p>The table below contains links to download the Apache SDAP (incubating) module source distributions as well as their signature and checksum files from ASF mirrors.</p>
 
 <h2>SDAP</h2>
+<h3>Latest Components</h3>
+<table class="version-table">
+    <thead>
+    <tr>
+        <th>Component</th>
+        <th>Release Version</th>
+        <th>Release Date</th>
+        <th>Source Distribution</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% assign latest = site.data.releases.releases.latest %}
+    {% for component in latest %}
+    <tr>
+        <td>{{ component.displayName }}</td>
+        <td>{{ component.release }}</td>
+        <td>{{ component.releaseDate }}</td>
+        <td>
+            <a href="https://www.apache.org/dyn/closer.lua/incubator/sdap/{{ component.release }}-incubating/apache-sdap-{{ component.component }}-{{ component.release }}-incubating-src.tar.gz">
+                [source] apache-sdap-{{ component.component }}-{{ component.release }}-incubating-src.tar.gz
+            </a>
+            <a href="https://www.apache.org/dyn/closer.lua/incubator/sdap/{{ component.release }}-incubating/apache-sdap-{{ component.component }}-{{ component.release }}-incubating-src.tar.gz.asc">
```

Comment:
Per [CalvinKirs](https://lists.apache.org/thread/r1om26omngvqjz1p2c763ttt3mndmmpk), these links should be to `downloads.apache.org` and not the mirror selector

Reply:
Ex: `https://downloads.apache.org/incubator/sdap/1.0.0-incubating/apache-sdap-ingester-1.0.0-incubating-src.tar.gz.asc`

Reply:
I marked the wrong lines. Should just be the .asc and .sha512 links that are updated. The .tar.gz links are fine

Reply:
Done